src/js/decafmud.socket.websocket.js

```diff
@@ -199,6 +199,9 @@ DecafWebSocket.prototype.onMessage = function(websocket, event) {
 	var reader = new FileReader();
 	reader.onload = function(e) {
 		var u8array = new Uint8Array(e.target.result);
+		this.decaf.socketData(u8array);
+		return;
```

Comment:
We should just delete the dead code following the return statement as it is never run

Reply:
Fixed, thanks! https://github.com/Tiger-Tom/DecafMUD/commit/09a046b77738f7d16d0bece69d55f47bc70bdf23